package.json

```diff
@@ -177,13 +186,16 @@
 		"gulp-check": "0.1.2",
 		"gulp-sftp": "0.1.5",
 		"husky": "2.3.0",
+		"i": "^0.3.6",
 		"jest": "24.7.1",
 		"jest-puppeteer": "4.1.1",
 		"lodash": "4.17.11",
 		"mocha": "6.1.4",
 		"mockery": "2.1.0",
 		"nock": "10.0.6",
 		"node-wp-i18n": "1.2.3",
+		"preact": "^8.4.2",
+		"preact-portal": "^1.1.3",
```

Comment:
Nit; note that dependencies in Jetpack are pinned (no `^`)

Reply:
Fixed!